docs/docs/reference/contextual/extension-methods.md

```diff
@@ -143,7 +143,7 @@ extension listOps on [T](xs: List[T]) {
   def third: T = xs.tail.tail.head
 }
 
-extension on [T](xs: List[T]) with Ordering[T] {
+extension on [T](xs: List[T])(using Ordering[T]) {
```

Comment:
Are these changes indented in the **not** new section ?

Reply:
No they are not. Thank for pointing it out!